reframe/frontend/printer.py

```diff
@@ -42,16 +46,22 @@ def status(self, status, message='', just=None, level=logging.INFO):
         else:
             status = status.ljust(self.status_width - 2)
 
+        status_stripped = status.strip().lower()
```

Comment:
I kept it because it was like this before, but I don't think we print anywhere the lowercase. Do you want to compare with uppercase statuses `[['FAIL', 'FAILED', 'ERROR']` for the color also?

Reply:
I'd say, better yes. I don't know what I had in mind when I put the call to `lower()`, but since we are using uppercase, we could simply avoid it.